src/init/features/dataconnect/index.ts

```diff
@@ -262,16 +262,17 @@ async function promptForService(
         info.serviceId = serviceName.serviceId;
         info.locationId = serviceName.location;
         if (choice.schema) {
-          if (choice.schema.primaryDatasource.postgresql?.cloudSql.instance) {
+          const primaryDatasource = choice.schema.datasources.find((d) => d.postgresql);
```

Comment:
nit: Can multiple datasources specify `postgresql`? Also given that the backend resource doesn't specify a "primary" datasource anymore -- is the "primary" datasource intended to just be the first in the list?

Maybe for now we can just pull the first datasource for now and verify it has a `postgresql` field?

Reply:
This is definitely a short sighted solution on my end, because we will definitely need a pretty major overhaul of the init code once we support multiple datasources.